### ISSUE TITLE
Improve badge details / search results for searches with multiple matches

### ISSUE THREAD

**Issue by Systemcluster** [badges/shields#9393]
:clipboard: **Description**

The search results on the site for searches with multiple results don't show any information to discern the results. It is impossible to figure out what each result is without clicking on it.

![inline results](https://github.com/badges/shields/assets/2847328/84db1b8c-2356-437f-8661-56f305cda941)
![results page](https://github.com/badges/shields/assets/2847328/f7fd7551-a054-4d4b-b8a2-afc573a2d954)

Even clicking on each result doesn't give much information what the badge is before filling in the fields and generating the badge.

![image](https://github.com/badges/shields/assets/2847328/6466c242-1939-4966-b1d8-35d8c1323797)

On the old website, I remember each badge having a description and example badge, making it easy to immediately see what a badge was and how it would appear. It would be very helpful having this information on the new site and in search results.

![shields io_](https://github.com/badges/shields/assets/2847328/695b83d7-a4e2-4144-a89c-9e609d438b44)

**Comment by Matt-Osborn** [comment]
Would the solution be to add `hover` functionality so that when you hoer over a search result you get additional details?

I'd be down to help implement this.

**Comment by ashuthe1** [comment]
I would love to work on this.

**Comment by chris48s** [comment]
Yeah agreed this is a problem. The biggest issue here is that we've got documentation pages with duplicate titles.

We should not have 4 or 5 different badges where the document title is "Crates.io"

Each one should have a distinct title e.g:

- "Crates.io downloads"
- "Crates.io version"
- "Crates.io license"
- etc

That is not where we are starting from, but there's not really any "UI coding" that can paper over that. It really requires a manual review process over the existing content, and that's pretty much the single biggest improvement that can be made.

The process of reviewing the way we define the documentation on services is underway, but moving slowly. Currently another round of review on https://github.com/badges/shields/pull/9320 is the blocker.

In parallel to that, if someone wanted to identify places where we've got >1 service with the same title and give them distinct titles (like I've shown above using Crates.io as an example), that would be a great contribution. I'd prefer to review a number of small PRs each tackling a handful of services where we have duplicates than one massive "boil the ocean" PR.

**Comment by my4ng** [comment]
This might be an unpopular opinion, but I actually much prefer the old website on this part. It is so much easier to browse all the options (rendered too) in a spreadsheet manner, and the new one is just counter-intuitive by forcing you to click on each sub-category without even a preview (though it is much better in terms of documentation). Is there any way to view the old '#your-badge' page at all?

**Comment by chris48s** [comment]
The equivalent of the `#your-badge` (static badge builder) would be https://shields.io/badges/static-badge

**Comment by my4ng** [comment]
@chris48s Sorry I should have been more clear: I meant like an example sheet, like this:

![](https://programmergirlsite.files.wordpress.com/2020/06/image-35.png?w=1024)

(Just an image of the old webpage from Google Images)

It is very clear what is contained and what each type looks like. Unfortunately, I couldn't find any decent way to do that now, other than to click each sub-category, where it doesn't even display the badge on the get-go, and you have to enter in some valid fields in order just to see what it looks like.

**Comment by uncenter** [comment]
Is there way we can bring search matches in titles to the top, prioritizing titles over content? In this example it would be great to move down the top four results to below the matches in titles:

<img width="250" alt="image" src="https://github.com/badges/shields/assets/47499684/d1a04a68-2467-4f7c-bd59-db1c58c89e71">

 This would be an upstream customization at https://github.com/easyops-cn/docusaurus-search-local/...
